lizard_neerslagradar/static/lizard_neerslagradar/lizard_neerslagradar.css

```diff
@@ -24,15 +24,27 @@ button#animate {
 }
 
 /* fix jquery ui slider */
+.ui-slider:after {
+    height: 1.4em;
+    margin-top: -4px;
+    width: 50%;
+    content: "";
+    float: right;
+    border-left: 2px solid rgba(0, 52, 89, 0.7);
+}
 .ui-slider .ui-slider-handle {
     background-color: #49AFCD;
     background-image: linear-gradient(to bottom, #5BC0DE, #2F96B4);
```

Comment:
Jij hebt hier niets aan gewijzigd, maar moet die `to bottom` geen quotes ofzo hebben?